trl/trainer/grpo_config.py

```diff
@@ -112,9 +112,8 @@ class GRPOConfig(TrainingArguments):
             set `sync_ref_model=True`.
 
         > Parameters that control the logging
-
         log_completions (`bool`, *optional*, defaults to `False`):
-            Whether to log the completions during training.
+            Print a sample of (prompt, completion) pairs every `logging_steps` steps. Also logs to `wandb` if available.
```

Comment:
```suggestion
        > Parameters that control the logging

        log_completions (`bool`, *optional*, defaults to `False`):
            Whether to log a sample of (prompt, completion) pairs every `logging_steps` steps. If `rich` is installed,
            it prints the sample. If `wandb` logging is enabled, it logs it to `wandb`.
```

Reply:
I had to change this manually, GitHub doesn't allow committing the suggestion for some reason.